compose.yml

```diff
@@ -1,10 +1,8 @@
-version: '3'
```

Comment:
For the version declaration is necessary. Otherwise the docker-compose complains and does not build.
Also, my docker-compose does not like the renaming.

Is that because I am running an old docker-compose?

Reply:
Did you run `npm install` in the root directory first?

Reply:
Interesting, according to the compose spec, the `version` flag is obsolete and doesn't really do anything: https://docs.docker.com/compose/compose-file/04-version-and-name/#version-top-level-element-obsolete.

But we can also add it back, doesn't really hurt...

Similarly, `compose.yml` is supposed to be the canonical name (https://docs.docker.com/compose/compose-application-model/#the-compose-file), but we can also go back to `docker-compose.yml` if it doesn't work for older docker-compose versions.

Reply:
I guess it's the difference between `docker-compose` and `docker compose` (the `docker-compose-plugin`). I installed the later and called it via `docker compose up` (so, without the `-` then it worked :) )

We should note this in the readme